contrib/docker/multi-stage-frontend/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:12 AS build
+
+# This dockerfile does not require the app to be built on the host machine.
+# simply builds backstage in node image then transfers compiled files to nginx container.
+
+# You can build this image with `docker build -f Dockerfile.standalone -t backstage .`
```

Comment:
Should update this line, or even get rid of this comment block since it has a README now

Reply:
Done.